plugins/nodes/src/js/columns/NodesTableGPUColumn.tsx

```diff
@@ -1,30 +1,40 @@
 import * as React from "react";
-import { compareValues } from "#PLUGINS/nodes/src/js/utils/compareValues";
+import sort from "array-sort";
 import Node from "#SRC/js/structs/Node";
 // TODO: DCOS-39079
 // import { IWidthArgs as WidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
 import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
 import { SortDirection } from "plugins/nodes/src/js/types/SortDirection";
 import { TextCell } from "@dcos/ui-kit";
 
+function getGpuUsage(data: Node): number {
+  return data.getUsageStats("gpus").percentage;
+}
+
 export function gpuRenderer(data: Node): React.ReactNode {
   return (
     <TextCell>
-      <span>{data.getUsageStats("gpus").percentage}%</span>
+      <span>{getGpuUsage(data)}%</span>
     </TextCell>
   );
 }
 
+function compareNodesByGpuUsage(a: Node, b: Node): number {
+  return getGpuUsage(a) - getGpuUsage(b);
+}
+
+function compareNodesByHostname(a: Node, b: Node): number {
+  return a
```

Comment:
Wonder if we need a `compareNodesByHostname ` Util function 🤔

Reply:
Thought about it, but I wanted to keep the scope of this as little as possible. If you add this as a utility you open up the discussion of why do we have it for hostnames and not the others.